helm/nostr/strfry-chart/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: strfry-chart
```

Comment:
Perhaps just "strfry". the "-chart" is probably unnecessary and gets littered into the default names for the kubernetes objects